CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+##  [Unreleased]
+
+### Changed
+
+- Removed `DeviceState` from `NetworkInterfaceConfig`
```

Comment:
I think this should be rephrased along the lines of "`PUT` operations on `/network-interfaces` API resources no longer accept the previously required `state` parameter.", because that's the externally visible effect.